cosipy/response/FullDetectorResponse.py

```diff
@@ -298,8 +298,8 @@ def _open_rsp(cls, filename, Spectrumfile=None,norm="Linear" ,single_pixel = Fal
 
             # If fast method fails, use long method, which should work in all cases.
             except:
-                print("Initial attempt failed.")
-                print("Using long method...")
+                logger.error("Initial attempt failed.")
```

Comment:
I think this should be info(). Despite the message, in this context, the algorithm is not really failing, it just couldn't do the fast method, which is fine. The problem with using error is that it will show up if the use decides to use the "error" logging level.